tests/MongoDB.Analyzer.Tests/Infrastructure/CodeBasedTestCasesSourceAttribute.cs

```diff
@@ -65,10 +65,10 @@ private DiagnosticTestCase[] CreateTestCases(MemberInfo memberInfo)
             from attribute in testCasesAttributes
             where EnvironmentUtilities.IsDriverTargetFrameworkSupported((Core.DriverTargetFramework)(int)attribute.TargetFramework)
             from version in DriverVersionHelper.FilterVersionForRange(attribute.Version)
-            group new DiagnosticRule(attribute.RuleId, $"{attribute.Message}_v{version.ToString("V", new VersionFormatter())}")
+            group new DiagnosticRule(RuleId: attribute.RuleId, Message: $"{attribute.Message}_v{version.ToString("V", new VersionFormatter())}", attribute.Location
+            )
```

Comment:
Let keep this one line (closing bracket)

Reply:
Done